### PR TITLE
Create: add tooltip to recommended funding cycle

### PR DESCRIPTION
**Pull request by wraeth-eth**
## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2483** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2483" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2483?utm_source=stack-comment).

File: src/components/Create/components/pages/FundingCycles/FundingCyclesPage.tsx
Comment (tomquirk) [inline src/components/Create/components/pages/FundingCycles/FundingCyclesPage.tsx:105]: **suggestion** might be nicer to wrap each text block in a `<p>`

(we should probably ban `br` in the codebase lol)

**Review by tomquirk — COMMENTED** [review]
yo coupla comments!

**Review by tomquirk — APPROVED** [review]

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2483?utm_source=stack-comment).


Job Reference: LMkvkXFiFBF1D050qgTf

**Pull request merged**
